routing/routing.go

```diff
@@ -371,6 +378,8 @@ func (r *Routing) startReceivingUpdates(o Options) {
 					}
 				}
 				r.log.Info("route settings applied")
+				r.metrics.UpdateGauge("routes.total", float64(len(rt.validRoutes)))
+				r.metrics.UpdateGauge("routes.updated_timestamp", float64(rt.created.Unix()))
```

Comment:
I think we may need to wrap this into nil check - existing codebases do not supply metrics instance

Reply:
done